webapp/lib/model.g.dart

```diff
@@ -44,11 +44,27 @@ class Conversation {
     };
   }
 
+  Future<bool> addToTagIds(DocPubSubUpdate pubSubClient, String newValue) {
```

Comment:
nit: I think `addTagId` sounds better? unless `newValue` can be something else other than a tag ID?

Reply:
Good point. Fixed.